modules/ROOT/pages/asm-secret-group-creation-task.adoc

```diff
@@ -21,7 +21,7 @@ The name of your secret group must:
 * Start with a letter
 * Contain only letters, numbers, and dashes (but cannot end with a dash)
 
-The secret group appears in the *Secret Groups* list view. Now you can add the necessary secret types.
+The secret group appears in the *Secret Groups* list view. You can now add the necessary secret types, for example, a keystore or a truststore. 
```

Comment:
What's the actual action? "...list view. Select a listed item (click a listed item) to add the necessary..." ?